requirements.txt

```diff
@@ -2,10 +2,10 @@ numpy<1.27
 cython
 scikit-learn==1.3.2
 git+https://github.com/pyRiemann/pyRiemann#egg=pyriemann
-qiskit==0.46.1
+qiskit==1.0.2
```

Comment:
Consider pinning to `==1.*`. One of the features of the 1.0 release is the stability guarantee that no versions of Qiskit 1.x will break compatibility. Same for the `setup.py`.